app/views/layouts/_width_container_wide.html.erb

```diff
@@ -2,6 +2,16 @@
   <div class="govuk-width-container govuk-width-container__wide">
     <%= yield(:before_content) %>
 
+    <div class="govuk-breadcrumbs">
+      <ol class="govuk-breadcrumbs__list">
+        <% breadcrumbs.each do |crumb| %>
+          <li class="govuk-breadcrumbs__list-item">
+            <%= crumb %>
+          </li>
+        <% end %>
+      </ol>
+    </div>
+
```

Comment:
I think this could probably be shortened as we already have the [govuk_breadcrumbs component](https://govuk-components.netlify.app/components/breadcrumbs/).

```suggestion
    <%= govuk_breadcrumbs(breadcrumbs:) %>
```